brave/browser-laptop
# Defer setAutofill*Detail to frame.js

fix #4350

Auditors: @bbondy

Test Plan:
1. open about:autofill from about:about
2. add/edit entry should work fine
